File: Python-Programing-Basics/Conditional Statements/Conditional Statements - Exercises/07_world_swimming_record/world_swimming_record.py
world_record_in_seconds = float(input())
distance = float(input())
swim_time_for_meter = float(input())

slow_down = distance // 15
time = distance * swim_time_for_meter + slow_down * 12.5

if time < world_record_in_seconds:
    print(f"Yes, he succeeded! The new world record is {time:.2f} seconds.")

else:
    print(f"No, he failed! He was {time - world_record_in_seconds:.2f} seconds slower.")

File: JavaScript/Programming Basics - JavaScript/Conditional Statements/worldSwimmingRecord.js
function calculateWorldRecord (input) {
    let worldRecord = Number(input[0]);
    let distanceInMeters = Number(input[1]);
    let timeInSecondsForOneMeter = Number(input[2]);
    let totalTime = distanceInMeters * timeInSecondsForOneMeter;
    totalTime += Math.floor(distanceInMeters / 15) * 12.5;
    if (worldRecord <= totalTime) {
        console.log(`No, he failed! He was ${(totalTime-worldRecord).toFixed(2)} seconds slower.`)
    } else {
        console.log(` Yes, he succeeded! The new world record is ${totalTime.toFixed(2)} seconds.`)
    }
}

calculateWorldRecord(["10464",
"1500",
"20"]);

calculateWorldRecord(["55555.67",
"3017",
"5.03"]);